### ISSUE TITLE
fetch and list submissions from instructors

### ISSUE THREAD

**Issue by aalmanza1998** [lxylxy123456/ngshare#35]
Currently instructors are not able to fetch and list submissions from instructors in ngshare. Instructors should be able to do this.

**Comment by lxylxy123456** [comment]
Can you give me a way to reproduce this issue?

**Comment by Lawrence37** [comment]
Is it just me, or do the commits not change the code logic?

**Comment by Lawrence37** [comment]
I ran into a problem with vngshare's database initialization. Instructors are not added as students, so I get "Student not found" for instructor submissions.

**Comment by lxylxy123456** [comment]
I think I found this bug in the code and committed to SOME branch, but I do not remember. Which API are you accessing? 
tl;dr: Instructors are not initialized as students; the logic should test "instructor or student", but it was only testing "student".

**Comment by Lawrence37** [comment]
I'm using get student submission (GET `/api/submission/<course_id>/<assignment_id>/<student_id>`).

**Comment by Lawrence37** [comment]
Fixed.
